modules/RTC/JitsiLocalTrack.js

```diff
@@ -96,6 +96,14 @@ export default class JitsiLocalTrack extends JitsiTrack {
             // Cache the constraints of the track in case of any this track
             // model needs to call getUserMedia again, such as when unmuting.
             this._constraints = track.getConstraints();
+
+            // Safari returns an empty constraints object, construct the constraints using getSettings.
+            if (!Object.keys(this._constraints).length && browser.isSafari() && videoType === VideoType.CAMERA) {
```

Comment:
Why the browser check?

Reply:
I wasn't sure if RN implemented getConstraints(), didn't want to mess it up when we start using the new gUM flow on RN as well. If it is implemented there, I can remove the browser check.

Reply:
Removing the old gUM flow is not a small change, IMHO that is better handled when that change happens.

Reply:
Sure, makes sense. I pushed the change with the browser check removed. Thanks for reviewing !